### PR TITLE
making the build compatible with heroku

### PR DESCRIPTION
**Pull request by erchaves**
This lets you run it on heroku.

Note: It won't work on heroku right out of the box though unless you allow some of the devDependencies to load. This can be done with:
`heroku config:set NPM_CONFIG_PRODUCTION=false`
...Or another build process could be added to postinstall. 

Not sure if it's worth adding a note about that...

Looking good so far my friend!

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,10 +2,15 @@
   "name": "example-app",
   "version": "0.0.0",
   "private": true,
+  "config" : {
+    "port" : 3000
+  },
   "scripts": {
-    "start": "DEBUG=example-app PORT=3000 NODE_ENV=production node bin/www",
+    "start": "DEBUG=example-app NODE_ENV=production node bin/www",
+    "clean": "rm -rf www",
```
Comment (jomaxx) [inline package.json:10]: where is this being used? could probably use it here https://github.com/jomaxx/example-app/pull/6/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R11

**Comment by jomaxx** [comment]
@erchaves this looks great, thanks for the update, my only suggestion is this https://github.com/jomaxx/example-app/pull/6#discussion_r46578420

**Comment by jomaxx** [comment]
@erchaves i think i'm gonna keep this outside of master, i want to keep this agnostic from deployments. this is heroku specific. but this looks good if you are trying to deploy to heroku

**Comment by erchaves** [comment]
Sounds good. I think it might also be needed for other hosting platforms that set their own process.env.PORT, but I'm not sure.

**Pull request closed without merging**